### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by derek-hwang27**
- What was changed?
  - Added more stuff to onboarding doc

- How was it changed? (put specific specs here)
  - more details
  - changed naming and added doc to main readme

- Any special notes?
  - n/a

@michl1001
@derek-hwang27

### CODE REVIEW

**Comment by derek-hwang27** [comment]
Documentation check fails since there is no backend folder, this is expected

File: admin/onboarding/onboard.md [hunk 1]
```diff
@@ -16,13 +25,46 @@ Updated 6/3/21
...
+
 ## Understanding iNeedA\<br>
 iNeedA\<br> (Group) is founded on the belief that everyone is an ally to everyone else. We value phycological safety and promote an environment where people are able to present ideas and expect to be given constructive criticism. We value individuals who are open about their struggles, are not shy about asking for help, and are constantly improving through constructive criticism. 
 
+## Understanding Eucalyptus
+Eucalyptus is a light-weight online bullet journal tool that allows users to perform rapid logging, prioritization, and more. Eucalyuptus supports three types of bullets: `note`, `task`, and `event`. Eucalyptus also supports the inclusion of additional entries in the form of plaintext documents or images. 
+
+Eucalyptus uses IndexDB as a database for storing and retrieving bullets. Our API supports storing bullets and retriving bullets based on queries (Date, Bullet Type, etc.)
+
+## Building Eucalyptus
```
Comment (michl1001) [inline admin/onboarding/onboard.md:37]: What does build mean in this context? Our code doesn't need any builds (local or repo)?

Comment (michl1001) [reply admin/onboarding/onboard.md:37]: The code gets auto-tested and such, but we don't use any major packages that need compiling or things like that. Deployment is it's own separate thing

File: admin/onboarding/onboard.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
 # iNeedA\<br> Onboarding Guide
-Updated 6/3/21
+Updated 6/6/21
 
 ## Navigation
 - [iNeedA\<br> Onboarding Guide](#ineedabr-onboarding-guide)
```
Comment (michl1001) [inline admin/onboarding/onboard.md:5]: We should add a guide of how we expect things to be contributed to the repo (what is the main PR process, where questions could be asked, etc)

File: admin/onboarding/onboard.md (hunk 1 above)
Comment (derek-hwang27) [reply admin/onboarding/onboard.md:37]: that's why they generate "automatically" and don't "require extra actions from the developer"

File: admin/onboarding/onboard.md (hunk 2 above)
Comment (derek-hwang27) [reply admin/onboarding/onboard.md:5]: addressed

**Pull request closed without merging**